prettier.config.mjs

```diff
@@ -0,0 +1,3 @@
+import baseline from '@mui/infra/prettier';
```

Comment:
Let's use named export? Also, would it be beneficial for the future to use factory functions as well?